guides/source/error_reporting.md

```diff
@@ -15,7 +15,7 @@ After reading this guide, you will know:
 Error Reporting
 ------------------------
 
-The Rails [error reporter](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html) provides a standard way to collect exceptions that occur in your application and report them to your preferred service or location.
+The Rails [error reporter](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html) provides a standard way to collect exceptions that occur in your application and report them to your preferred service or location (e.g. a monitoring service such as [Sentry](https://github.com/getsentry/sentry-ruby/blob/e18ce4b6dcce2ebd37778c1e96164684a1e9ebfc/sentry-rails/lib/sentry/rails/error_subscriber.rb)).
```

Comment:
It doesn't feel clear what we're showing an example of.  Perhaps we could explain in a single sentence what that example is? Something like:

```suggestion
The Rails [error reporter](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html) provides a standard way to collect exceptions that occur in your application and report them to your preferred service or location (e.g. you could report the errors to a monitoring service such as Sentry [using a subscriber](https://github.com/getsentry/sentry-ruby/blob/e18ce4b6dcce2ebd37778c1e96164684a1e9ebfc/sentry-rails/lib/sentry/rails/error_subscriber.rb)).
```

I'm not sure if the above is accurate or was your intention so update as you feel necessary.